src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/Internal/TypesTab/TypesTab.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../../../../../Models/Services/StoryUtilities';
+import TypesTab from './TypesTab';
+import { ITypesTabProps } from './TypesTab.types';
+
+const wrapperStyle = { width: '100%', height: '600px', padding: 8 };
+
+export default {
+    title: 'Apps/Legion/TypesTab',
+    component: TypesTab,
+    decorators: [getDefaultStoryDecorator<ITypesTabProps>(wrapperStyle)]
+};
+
+type TypesTabStory = ComponentStory<typeof TypesTab>;
+
+const Template: TypesTabStory = (args) => {
+    return <TypesTab {...args} />;
```

Comment:
do you think we should have stories for the tabs? or just have stories for the wizard and interactions that navigate you to the right tab? Then we don't have to set up all the contexts and mock data in both places.

Reply:
I thought about that, I think I'll remove the tab stories, never really use them.